snapshots/lcow/lcow.go

```diff
@@ -321,7 +325,18 @@ func (s *snapshotter) createSnapshot(ctx context.Context, kind snapshots.Kind, k
 			return nil, err
 		}
 
-		scratchSource, err := s.openOrCreateScratch(ctx)
+		var snapshotInfo snapshots.Info
+		for _, o := range opts {
+			o(&snapshotInfo)
+		}
+
+		var sizeGB int
+		if sizeGBstr, ok := snapshotInfo.Labels[rootfsSizeLabel]; ok {
+			i32, _ := strconv.ParseInt(sizeGBstr, 10, 32)
```

Comment:
just curiosity,  why not return error here?

Reply:
Good question. I guess we should if someone specifies a non-integer value. I will update.

Reply:
Thanks!